### PR TITLE
Lesson 2 - home work

### PR DESCRIPTION
**Pull request by DergachovaAnna**
Finished home work tasks

### CODE REVIEW

File: lesson_2.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# TASK 2: You have 2 companies with people. One of the companies, let it be Eleks, was taken over by Toshiba.
+# Show it in code. Keep in mind that people with the same name can be in both companies.
+
+toshiba_personnel = ["Ann", "Mary", "Jim", "Ashley", "Garry", "James"]
+eleks_personnel = ["Margo", "Andrey", "Billy", "James", "Ann"]
+
+new_company_list = toshiba_personnel.copy()
+new_company_list.extend(eleks_personnel)
+print(new_company_list)
+
```
Comment (bohdandovbsyh) [inline lesson_2.py:11]: eleks still have employees

File: lesson_2.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# "John Dow", "John Dow", "Marta Dow" => "John Dow", "Marta Dow "
+
+mixed_list = ["Maggy Smith", "John Doe", "John Doe", "Jane Smith", "Maggy Smith", "Bill Johns", "Amely Doe"]
+unique_list = []
+
+for name in mixed_list:
+    if name in unique_list:
+        continue
+    else:
+        unique_list.append(name)
```
Comment (bohdandovbsyh) [inline lesson_2.py]: it is HW for build-in functions. You can do this task without loops

File: lesson_2.py (hunk 1 above)
Comment (DergachovaAnna) [reply lesson_2.py:11]: done

File: lesson_2.py (hunk 2 above)
Comment (DergachovaAnna) [reply lesson_2.py]: done

**Pull request merged**